content/en/docs/security/otel-collector-configuration-best-practices.md

```diff
@@ -0,0 +1,139 @@
+---
+title: OpenTelemetry Collector Configuration Best Practices
+linkTitle: Collector config
+description:
+  Follow best practices to securely configure OpenTelemetry Collector.
+weight: 20
+---
+
+When setting up configuration for OpenTelemetry (OTel) Collector, consider the
+following practices to better secure your OTel Collector instance.
+
+<!--
+TODO: SHOULD only enable the minimum required components. What are those?
+--->
+
+## Receivers and Exporters
+
+- Receivers and Exporters can be either push or pull-based. In either case, the
+  connection established SHOULD be over a secure and authenticated channel.
+- Unused receivers and exporters SHOULD be disabled to minimize the attack
+  vector of the Collector.
+- Receivers and Exporters MAY expose buffer, queue, payload, and/or worker
+  settings via configuration parameters. If these settings are available, users
+  should proceed with caution before modifying the default configuration values.
+  Improperly setting these values may expose the OpenTelemetry Collector to
+  additional attack vectors including resource exhaustion.
```

Comment:
```suggestion
- Receivers and exporters may expose buffer, queue, payload, and worker
  settings using configuration parameters. If these settings are available, you
  should proceed with caution before modifying the default configuration values.
  Improperly setting these values may expose the OpenTelemetry Collector to
  additional attack vectors including resource exhaustion.
```

Best going with second person rather than addressing "users" if the guide is aimed at folks who can implement OTel security on their own.

Reply:
Sounds good. Fixed. This is a good improvement for what I copied and pasted from the source document.